cobc/scanner.l

```diff
@@ -323,6 +324,14 @@ AREA_A		"#AREA_A"\n
 	cobc_areacheck = 0;
 }
 
+<*>^[ ]?"#INCLUDE".*\n {
+	yytext[strlen(yytext)-1] = '\0';
```

Comment:
instead of dropping te last position, just use `.*/\n` instead to not include that in yytext